src/visualization/types/SimpleTable/PagedTable.tsx

```diff
@@ -42,7 +42,7 @@ const measurePage = (
   let lastSignature
   let signature
 
-  while (rowIdx <= result.table.length) {
+  while (rowIdx < result.table.length) {
```

Comment:
nice. does this fix the 2 records on the last page issue?

Reply:
Yes. The before and after images has to do with that issue.